src/test/store/receive-profile.test.js

```diff
@@ -967,14 +950,12 @@ describe('actions/receive-profile', function () {
         getProfileFromTextSamples('0xA[lib:libxul]');
       unsymbolicatedProfile.meta.symbolicated = false;
 
-      window.fetch = jest.fn().mockResolvedValue(
-        (({
-          ...fetch200Response,
-          json: () => Promise.resolve(), // Used when fetching symbols.
-          arrayBuffer: () =>
-            Promise.resolve(encode(serializeProfile(unsymbolicatedProfile))),
-        }: any): Response)
-      );
+      window.fetch
+        .get(
+          'https://storage.googleapis.com/profile-store/FAKEHASH',
+          makeProfileSerializable(unsymbolicatedProfile)
+        )
+        .post('https://symbolication.services.mozilla.com/symbolicate/v5', {});
```

Comment:
This is needed because we are triggering the symbolication right?

Reply:
Yeah. Without this line we would hit the catch defined above that is a `403`, and we'd get an error about the fact that the result can't be decoded as JSON.

The previous code was "cheating" and depending on the impementation details, that symbolication is using `json()` but profile fetching is using `arrayBuffer()`. So here I'm essentially doing something similar than before by returning an empty object, and I get the same error as before.